src/libsync/account.cpp

```diff
@@ -181,6 +189,9 @@ void Account::setCredentials(AbstractCredentials *cred)
     if (jar) {
         _am->setCookieJar(jar);
     }
+    if (hadProxy) {
```

Comment:
isn't `hadProxy` a proxy value (ah!) for `proxy.type() != QNetworkProxy::NoProxy`?

Reply:
actually `QNetworkProxy::DefaultProxy` after some digging and testing ;)